### PR TITLE
Export or run a SimpleLanguage program.

### PR DESCRIPTION
**Pull request by snim2**
Here is a [simple example](https://youtu.be/1KiCJSFz9co)

### CODE REVIEW

File: lib/eco/treemanager.py [hunk 1]
```diff
@@ -1427,14 +1427,40 @@ def export(self, path=None, run=False):
...
+            if not os.environ.has_key("GRAAL_WORKSPACE"):
+                sys.stderr.write("GRAAL_WORKSPACE environment not set")
+                return
+            working_dir = os.path.join(os.environ["GRAAL_WORKSPACE"], "graal-compiler")
+            f = tempfile.mkstemp(suffix=".sl")
+            self.export_as_text(f[1])
+            # Run this command:
+            #     $ cd $GRAAL_WORKSPACE/graal-compiler
+            #     $ ../../mx/mx --vm graal sl sl_hello_world.sl
+            return subprocess.Popen(["../../mx/mx", "--vm", "graal", "sl", f[1]],
```
Comment (ltratt) [inline lib/eco/treemanager.py:1456]: Previous (bitter) experience suggests we should be passing MX in with an environment variable (or similar).

Comment (snim2) [inline lib/eco/treemanager.py:1456]: `$GRAAL_WORKSPACE` should be the whole workspace which holds all the compilers and tools, so it should contain `mx`.

Comment (ltratt) [inline lib/eco/treemanager.py:1456]: Right, but the `../../mx/mx` path isn't relative to `$GRAAL_WORKSPACE_ in the`Popen` call.

Comment (ltratt) [inline lib/eco/treemanager.py:1456]: I'm wrong! Because of `cwd`.

Comment (snim2) [inline lib/eco/treemanager.py:1456]: No, the `Popen()` call is passed a `cwd` argument which sets its working directory.

Comment (ltratt) [inline lib/eco/treemanager.py:1456]: Yep, I'd spotted I'd got that wrong. So ignore me :)

**Comment by ptersilie** [comment]
Looks good. I like the youtube videos. :) And that I can hear your pc fans speeding up when you run the second example. ;)

**Pull request merged**